packages/adminrouter/extra/src/Makefile.common

```diff
@@ -68,14 +69,14 @@ devkit:
 				/bin/bash -c "\$$OPENRESTY_COMPILE_SCRIPT" && \
 		docker commit $$(docker ps -a -q -f name=$(DEVKIT_NAME)) \
 			mesosphere/$(DEVKIT_NAME):full && \
-		docker rm $(DEVKIT_NAME) && \
+		docker rm -f $(DEVKIT_NAME) && \
 		docker rmi -f mesosphere/$(DEVKIT_NAME):noresty; \
 	fi
 
 .PHONY: update-devkit
 update-devkit: clean-devkit-container
 	@# Let's suppress ALL the output:
```

Comment:
❓ Existing: This says that all output will be suppressed, but `docker commit` is not.

Reply:
It does not have to be explicitly silenced as it is linked to the previous statement with the `&&` and for the `make` command this two count as one.